src/main.zig

```diff
@@ -1125,7 +1126,12 @@ fn buildOutputType(
                         try clang_argv.appendSlice(it.other_args);
                     },
                     .positional => {
-                        const file_ext = Compilation.classifyFileExt(mem.spanZ(it.only_arg));
+                        var file_ext: Compilation.FileExt = undefined;
```

Comment:
Style nit, don't initialize variables in invalid state. You can easily use a block or, if possible, the `if` expression to initialize it:
```zig
const file_ext = if (treat_as_file_ext == .unknown) X else Y;
// or
const file_ext = blk: { if (treat_as_file_ext == .unknown) break :blk X else break :blk Y };
```